h2o-core/src/main/java/water/persist/PersistManager.java

```diff
@@ -429,6 +436,8 @@ public long length(String path) {
       validateHdfsConfigured();
       long l = I[Value.HDFS].length(path);
       return l;
+    } else if (isGcsPath(path)) {
```

Comment:
I think these if-statements are good for now but we should eventually refactor the code and make adding new persist backends pluggable.

Reply:
Totally agree. Didn't want to struggle with that in this PR.